### DISCUSSION TITLE
Modular Monolith with DDD for "Orders" domain that is well designed and is finished

### DISCUSSION THREAD

**Discussion by jflaga** [kgrzybek/modular-monolith-with-ddd#317] [Q&A]
In the section "1.3 Reason" in README, there is this statement:

> The reason for creating this repository is the lack of something similar. Most sample applications on GitHub have at least one of the following issues:
> ...
> - Implements "Orders" domain - yes, everyone knows this domain, but something different is needed

May I ask for links to repositories which implements Modular Monolith with DDD for "Orders" domain, and is well designed and is finished?

Thanks.

**Comment by forgoty** [comment]
Hello!

There is [eShop](https://github.com/dotnet/eShop) maintained by dotnet team itself. Check this out.
